jukebox_radio/music/search.py

```diff
@@ -40,29 +44,14 @@ def get_search_results(user, provider_slug, query, formats):
         # TRACK
         if search_result["format"] in ["track", "video"]:
             tracks.append(
-                Track(
```

Comment:
Nice little improvement. This substitution is ok because there is no need to manually enumerate - all can be passed through.